intera_interface/src/intera_interface/camera.py

```diff
@@ -175,26 +187,116 @@ def start_streaming(self, camera_name):
 
     def stop_streaming(self, camera_name):
         """
-        Stop camera streaming by given the camera name.
+        Stop camera streaming for the given camera name.
 
         @type camera_name: str
         @param camera_name: camera name
 
         @rtype: bool
-        @return: False if camera not exists in camera name list or not able
-                 to stop streaming camera. True if the camera not is streaming
-                 mode or the camera successfully stop streaming.
+        @return: False if camera does not exist in camera name list or not able
+                 to stop streaming camera. True if the camera is already not in
+                 streaming mode or streaming is successfully stopped.
         """
         if not self.verify_camera_exists(camera_name):
             return False
         elif self._camera_streaming_status(camera_name):
             self.cameras_io[camera_name]['interface'].set_signal_value(
                 "camera_streaming", False)
             if self._camera_streaming_status(camera_name):
-                rospy.logerr(' '.join(["Failed to stop", camera_name,
+                rospy.logerr(' '.join(["Failed to stop ", camera_name,
                 " from streaming on this robot."]))
                 return False
             else:
                 return True
         else: # Camera not in streaming mode
             return True
+
+    def get_exposure(self, camera_name):
+        """
+        Return the current exposure setting for the given camera.
+
+        @type camera_name: str
+        @param camera_name: camera name
+
+        @rtype: int
+        @return: current exposure setting (-1 means auto-exposure was set)
+        """
+        if self.verify_camera_exists(camera_name):
+            return self.cameras_io[camera_name]['interface'].get_signal_value('set_exposure')
```

Comment:
This will only return if the camera exists? I think we need an else `None` or -2 or something

Reply:
Sure, I can add an explicit return of `None`. It will return None anyways implicitly, but you're right - better to be explicit.